base/sort.jl

```diff
@@ -1482,8 +1482,9 @@ InitialOptimizations(next) = SubArrayOptimization(
 `DefaultStable` is an algorithm which indicates that a fast, general purpose sorting
 algorithm should be used, but does not specify exactly which algorithm.
 
-Currently, it is composed of two parts: the [`InitialOptimizations`](@ref) and a hybrid of
-Radix, Insertion, Counting, Quick sorts.
+Currently, when sorting short NTuples, this is an unrolled mergesort, and otherwise it is
```

Comment:
Should there be a compat notice indicating what versions of Julia support `NTuple` sorting?

Reply:
Yes, though it should live in `?sort`, not `?Base.Sort.DEFAULT_STABLE`